pkg/networkmanager/networking.css

```diff
@@ -268,21 +268,47 @@ h1 .onoff-ct {
     }
 }
 
-#zones-listing,
-#services-listing {
-    margin: 2rem 0;
+#zones-listing {
+    margin: 2rem 3rem;
 }
 
-#services-listing .listing-ct-body h5 {
-    margin: 1.5rem 0 1rem;
+#firewall-heading {
+    background: #FFFFFF;
+    padding: 10px;
 }
 
-#services-listing .listing-ct-body ul {
-    padding-left: 1rem;
+#firewall-heading > h1 {
+    display: flex;
+    align-items: stretch;
+    justify-content: space-between;
+}
+#firewall-heading > h1 > span:first-child {
+    display: flex;
 }
 
-#services-listing .listing-ct-body li + li {
-    margin-top: 1rem;
+#firewall-heading .btn-group {
+    display: inline-flex;
```

Comment:
adding `align-items: center;` will help the button not try to grow to match the height of it's parent and will keep it the same height as the other buttons on the page.